platform/lumin-runtime/elements/builders/grid-layout-builder.js

```diff
@@ -14,7 +14,9 @@ export class GridLayoutBuilder extends PositionalLayoutBuilder {
     }
 
     create(prism, properties) {
-        this.throwIfInvalidPrism(prism);
+        // TODO(tcuadra): Should LayoutBuilder inherit from UiNodeBuilder?
```

Comment:
I'd like to resolve this in this review before merging. The simplest change is to just move throwIfInvalidPrism up to the first common ancestor (ElementBuilder). If it makes more sense to change the inheritance tree, though, I can do that instead (this worked in practice, at least with my Tic Tac Toe example)